pyramid_jinja2/i18n.py

```diff
@@ -18,10 +18,11 @@ def localizer(self):
         except AttributeError: # pragma: nocover (pyramid < 1.5)
             return i18n.get_localizer(request)
 
-    def gettext(self, message):
+    def gettext(self, message, **mapping):
```

Comment:
Please change this to `mapping` to avoid future compatibility issues when we want to add other arguments like domain/context.

Also, shouldn't this be added to `ngettext` as well?

Reply:
@mattias-lidman would you please update your PR as requested?